examples/connext_dds/group_coherent_presentation/py/GroupCoherentExample_subscriber.py

```diff
@@ -155,7 +155,7 @@ def main():
     try:
         run_subscriber_application(args.domain, args.count)
     except Exception as e:
-        print("Exception in run_subscriber_application(): ", e)
```

Comment:
Actually this wasn't bad or necessarily inconsistent. I got a bit overzealous :)

Reply:
Would you like me to revert it back to that print statement?

Reply:
No, this is fine too :)